packages/camera/camera_platform_interface/pubspec.yaml

```diff
@@ -3,7 +3,7 @@ description: A common platform interface for the camera plugin.
 homepage: https://github.com/flutter/plugins/tree/master/packages/camera/camera_platform_interface
 # NOTE: We strongly prefer non-breaking changes, even at the expense of a
 # less-clean API. See https://flutter.dev/go/platform-interface-breaking-changes
-version: 1.5.0
+version: 1.5.1
```

Comment:
Nit: you are adding functionality, not fixing a bug, so semver calls for incrementing the second number, not the third.